featurebyte/routes/registry.py

```diff
@@ -118,6 +119,10 @@
 app_container_config = AppContainerConfig()
 
 # Register classes - please keep sorted by alphabetical order.
+app_container_config.register_class(AllDeploymentController)
+app_container_config.register_class(AllDeploymentService)
+app_container_config.register_class(AllCatalogService)
```

Comment:
nit: `AllCatalogService` can be before `AllDeployment*` for alphabetical ordering